lib/rules/jsx-no-target-blank.js

```diff
@@ -15,7 +15,7 @@ module.exports = function(context) {
         var relFound = false;
         var attrs = node.parent.attributes;
         for (var idx in attrs) {
```

Comment:
I looked at this today.  There is a `break` inside the `for` loop which doesn't work in the `forEach` version.  I'm sure there's a way around that, but I think this should be in a separate PR.  I'd like to have this one considered done and merged (if acceptable).